winit-core/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+#[macro_use]
```

Comment:
Nit: A small description at the top-level would be nice here.

Maybe something like:
```
//! # Core types for Winit
//!
//! Platform-agnostic types and traits useful when implementing Winit backends,
//! or otherwise interfacing with Winit from library code.
//!
//! See the [`winit`] crate for the full user-facing API.
//!
//! [`winit`]: https://docs.rs/winit
```